types/src/keyless/circuit_testcases.rs

```diff
@@ -63,8 +66,8 @@ pub(crate) static SAMPLE_JWT_PAYLOAD_JSON: Lazy<String> = Lazy::new(|| {
             {}
             "locale":"en",
             "iat":1700255944,
-            "exp":2700259544,
-            "nonce":"{}"
+            "nonce":"{}",
```

Comment:
cc @heliuchuan, not sure how you succeeded in generating a proof in the past for this since there was no comma after `nonce`...